runtime/src/launch/amudprun/launch-amudprun.c

```diff
@@ -97,13 +97,34 @@ static char** chpl_launch_create_argv(const char *launch_cmd,
     const char* ev_use_lldb = getenv("CHPL_COMM_USE_LLDB");
 
     if (ev_use_gdb != NULL || ev_use_lldb != NULL) {
+
+      // determine the terminal emulator to use
+      char* dbg_term = getenv("CHPL_COMM_DBG_TERM");
+      if(dbg_term == NULL) {
+        dbg_term = "xterm";
+      }
+      else if(strcmp(dbg_term, "xterm") != 0  && 
+              strcmp(dbg_term, "urxvt") != 0) {
+        // currently limiting to xterm and urxvt because of lack of
+        // testing/interest. Most other terminal emulators seem to support the
+        // -e flag, but there are some (like tilda) that uses different flags
+        // (-c for tilda). Ideally, there should be a lookup table for those
+        // flags if there is interest in adding support for more terminal
+        // emulators
+        chpl_warning("CHPL_COMM_USE_(G|LL)DB can only be set to xterm or urxvt",
```

Comment:
This should be `CHPL_COMM_DBG_TERM` instead of `CHPL_COMM_USE_(G|LL)DB`.

Reply:
Oops! Fixed it.